app/i18n/translations/dev.json

```diff
@@ -214,10 +215,12 @@
   "notifications.type.send": "Saída",
   "notifications.type.ticket": "Tíck",
   "notifications.type.transfer": "Transferência",
+  "passphrase.info.header": "Private passphrase",
+  "passphrase.info.message": "This passphrase ensures that no funds may be spent from this wallet without it.  It is used to encrypt your private keys associated with your addresses.  These private keys are required to sign transactions that spend from the addresses.  Make sure to use a high-security passphrase and keep this passphrase in a secure place.   With your wallet file and this private passphrase an attacker would have full control over any funds associated with this wallet.",
```

Comment:
I don't know if that's needed but you always have 2 spaces at the beginning of a sentence after a `.`

Reply:
The string on dev.json is copied from the source string (in this particular case, that string is in PassphraseInfoModal, last modified by ayp) so we'd need to check with him.

But really, this was probably just a typo... :)